src/Util/Validation.js

```diff
@@ -0,0 +1,199 @@
+import getMonths from "./getMonths";
+
+function validateFirstName(firstName) {
+    var validation = {
+        valid: true,
+        message: ""
+    }
+
+    if (!firstName || firstName.trim().length == 0) {
+        validation.message = "First Name is required"
+        validation.valid = false;
+    }
+
+    var regFirstName = /^[A-Za-z0-9\s\-]*$/g
+    if (!regFirstName.test(firstName)) {
+        validation.message = "First Name cannot contain special characters"
+        validation.valid = false;
+    }
+
+    return validation
+}
+
+function validateLastName(lastName) {
+
+    var validation = {
+        valid: true,
+        message: ""
+    }
+
+    if (!lastName || lastName.trim().length == 0) {
+        validation.message = "Last Name is required"
+        validation.valid = false;
+    }
+
+    var regLastName = /^[A-Za-z0-9\s\-]*$/g
+    if (!regLastName.test(lastName)) {
+        validation.message = "Last Name cannot contain special characters"
+        validation.valid = false;
+    }
+
+    return validation
+}
+
+function validateEmail(email) {
+    var validation = {
+        valid: true,
+        message: ""
+    }
+
+    if (!email || !email.includes("@") || !email.includes(".")) {
```

Comment:
this would enable input of invalid emails, like `ridvan@appa@hotmail.com`.
Why not use regex to validate email as well